### PR TITLE
[SRVCOM-891] serverless-operator nightly on 4.3 and 4.4

### PR DESCRIPTION
**Pull request by mgencur**
Depends on https://github.com/openshift-knative/serverless-operator/pull/376 as that PR brings the right Makefile target.

### CODE REVIEW

**Comment by nak3** [comment]
This looks good to me. 
But just in case, I would like to highlight this for @markusthoemmes @rhuss @aliok if any comments.

**Comment by mgencur** [comment]
/hold

https://github.com/openshift/ci-tools/pull/1005 has been merged so it should be possible to add Slack notifications. I'll do that in a minute.

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: api.ci
-  cron: 0 */6 * * *
+  cron: 0 */12 * * *
```
Comment (rhuss) [inline ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml:82]: Not sure if this make sense, but wouldn't it be better to choose a slightly different timeslot for each run, in order to stretch a bit the load on the infrastructure ? (if its running with elastic resources anyway, then of course this doesn't matter).

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml [hunk 2]
```diff
@@ -79,7 +79,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
```
Comment (rhuss) [inline ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml:80]: Is this job then still needed ? Looks like a duplication of openshift-knative-serverless-operator-master__4.3.yaml

**Comment by rhuss** [comment]
I see, upgrade tests are now part of the "all" make target.

How would the log file look like ? Would it be an even larger log output with all tests concatenated ?  or are there separate log file per test (e2e, upgrade ...)

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml (hunk 1 above)
Comment (mgencur) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml:82]: I'd say these few jobs will not make a difference among the large pool of tests from the whole OpenShift team.

**Comment by mgencur** [comment]
> I see, upgrade tests are now part of the "all" make target.
> 
> How would the log file look like ? Would it be an even larger log output with all tests concatenated ? or are there separate log file per test (e2e, upgrade ...)

The will be logged all together, but there are just a few tests from the Upgrade test suite (~10). Overall there are almost 400 tests.

**Comment by mgencur** [comment]
The tests on OCP 4.5 will still run in pre-submit (serving, eventing, upgrade. etc.) like they do now. I just modified the periodic job for 4.5 to run less frequently. It was already there with 6-hour cadence. We're only moving tests on OCP 4.3 and 4.4 to periodics.

File: ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml (hunk 2 above)
Comment (mgencur) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml:80]: It's needed. Yes. The file `openshift-knative-serverless-operator-master__4.3.yaml` is a complement of this. It includes images that should be built etc.

Comment (mgencur) [reply ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-master-periodics.yaml:80]: And this stuff on line 80 which you referenced is actually generated from that file.

**Pull request merged**